<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>High</class>
 <widget class="QWidget" name="High">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/sample/photo/High_pic/High_background.jpg);</string>
  </property>
  <widget class="QLabel" name="beatnumbers">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sample/photo/startbutton.jpg);</string>
   </property>
   <property name="text">
    <string>beatnumbers</string>
   </property>
  </widget>
  <widget class="QLabel" name="beatlevels">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>170</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sample/photo/startbutton.jpg);</string>
   </property>
   <property name="text">
    <string>beatlevels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sample/photo/startbutton.jpg);</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sample/photo/startbutton.jpg);</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLable1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sample/photo/High_pic/High_beatnumbers/High_beatnumbers_1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLable2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>300</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/sample/photo/High_pic/HIgh_beatlevels/High_beatlevels_1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
